CHANGELOG.md

```diff
@@ -71,6 +71,7 @@
 * [BUGFIX] Frontend, Query-scheduler: allow querier to notify about shutdown without providing any authentication. #4066
 * [BUGFIX] Querier: fixed race condition causing queries to fail right after querier startup with the "empty ring" error. #4068
 * [BUGFIX] Compactor: Increment `cortex_compactor_runs_failed_total` if compactor failed compact a single tenant. #4094
+* [BUGFIX] Purger: fix `Invalid null value in condition for column range` caused by `nil` value in range for WriteBatch query. #4128
```

Comment:
We've just cut a new Cortex version. Can you rebase `master` and move this to the top, under "master / unreleased"?

Reply:
Done and done.